server/aap/macros/pollution_report.py

```diff
@@ -49,7 +49,14 @@ def generate_pollution_story(item, **kwargs):
     # the regional value or just the site value, regional value rows seem to also contain a site value AQI
     # The results are stored into the values map
     url = 'https://airquality.environment.nsw.gov.au/aquisnetnswphp/getPage.php?reportid=1'
-    page = requests.get(url)
+    tries = 0
+    while tries < 5:
+        try:
+            page = requests.get(url, timeout=5, verify=False, allow_redirects=True)
+            break
+        except Exception as ex:
+            logger.exception(ex)
+            tries += 1
```

Comment:
The exception is never re-raised if the request still fails after 5 attempts.

Suggest checking here if tries == 5 then re-throw the exception